Assets/MRTK/StandardAssets/packagetemplate.json

```diff
@@ -24,7 +26,5 @@
     "bugs": {
         "url": "https://github.com/microsoft/MixedRealityToolkit-Unity/issues"
       },
-    "dependencies": {
-        "com.microsoft.mixedreality.toolkit.foundation": "%version%"
-    }
+    "dependencies": { }
```

Comment:
Do we need this field since it's empty?

Reply:
probably not. i have just been leaving it in for clarity

Reply:
removed